backend/apps/monitoring/models.py

```
@@ -20,3 +20,34 @@ class Meta:
     def __str__(self):
         return f"Backup {self.backup_timestamp} - {self.status}"
+
+
+class TaskRun(models.Model):
+    STATUS_CHOICES = [
+        ("PENDING", "Pending"),
+        ("STARTED", "Started"),
+        ("SUCCESS", "Success"),
+        ("FAILURE", "Failure"),
+        ("RETRY", "Retry"),
+    ]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)